response_operations_ui/templates/account/activate-account.html

```diff
@@ -63,35 +75,46 @@
                 {% endfor %}
                 {{
                     onsList({
+                        "element": 'ol',
                         "classes": "list--bare",
                         "itemsList": errorData
                     })
                 }}
             {% endcall %}
         {% endif %}
+            {% if errorData %}
+                </br>
+            {% endif %}
             <h1>Activate your account</h1>
             <form method="post" action="" class="form" role="form">
                 <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-              <span>An account has been created for <strong>{{ username }}</strong>.</span>
-              <p>To activate the account, you must create a password.</p>
-                {{
-                    onsPanel({
-                        "body": '<p>Your password must have:</p><ul>
-                        <li>at least 12 characters</li>
-                        <li>at least 1 uppercase letter</li>
-                        <li>at least 1 symbol (eg: ?!£%)</li>
-                        <li>at least 1 number</li></ul>'
-                    })
-                }}
+              <span>An account has been created for <strong>{{ username }}</strong>. 
+                To activate the account, you must create a password.</span>
+              <!--Double <br> required to ensure correct spacing between above span and password information section-->
+              </br>
```

Comment:
Same as above, using multiple <br> for spacing feels off to me.

Reply:
As above